bls/src/main/java/tech/pegasys/teku/bls/BLS.java

```diff
@@ -65,6 +69,10 @@ public static BLSSignature sign(BLSSecretKey secretKey, Bytes message) {
    * @return True if the verification is successful, false otherwise.
    */
   public static boolean verify(BLSPublicKey publicKey, Bytes message, BLSSignature signature) {
+    if (!BLSConstants.VERIFICATION_ENABLED) {
```

Comment:
This may be too pernickety, but I'd much rather see the constant named `VERIFICATION_DISABLED`. This for a couple of reasons:
  - It's easy to lose/forget the `!` at the start. It's an extra cognitive step (got to work out "not false"), and I've seen really stupid errors arise as a result of simple things like this.
  - The normal case is "enabled", flagging `DISABLED` in big letters shows more clearly that this is an exceptional path.

Reply:
I think it makes a lot of sense.

Reply:
Done.